templates/dev.rake

```diff
@@ -1,4 +1,4 @@
-if Rails.env.development? || Rails.env.test?
+unless Rails.env.production?
```

Comment:
Thanks for the discussion! I'll go ahead and change this PR.

Are you suggesting Staging uses the same DB instance as Production? Or were you referring to different types of databases?

So you're suggesting we add a different ENV variable for distinguishing staging from production, and code that depends on it? Any ideas what to call it? `HEROKU_ENV`?